llvm/lib/Transforms/Vectorize/VPlanUtils.cpp

```diff
@@ -124,3 +125,12 @@ bool vputils::isUniformAcrossVFsAndUFs(VPValue *V) {
         return false;
       });
 }
+
+VPBasicBlock *vputils::getTopLevelVectorLoopHeader(VPlan &Plan,
```

Comment:
```suggestion
VPBasicBlock *vputils::getFirstLoopHeader(VPlan &Plan,
```
as it is agnostic to loop being vector or not, can also be FirstTopLevel but First implies TopLevel?